phpunit/custom-build.xml

```diff
@@ -9,7 +9,7 @@
   <target name="phpunit.unit.custom" depends='plugin-fusioninventory.prepare' description="Run unit tests with PHPUnit">
    <phpunit.run type="custom" configuration='${fusioninventory.directory}/phpunit/testsuites_integration.xml'>
      <testfile>
-       <arg path='${tests.directory}/2_Integration/SoftwareVersionUpdateTest.php' />
+       <arg path='${tests.directory}/1_Unit/PrinterUpdateTest.php' />
```

Comment:
You should avoid changes on this file... :/

Maybe add it to your local excludes (`echo "phpunit/custom-build.xml" >> .git/info/exclude`)